bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-edit.vue

```diff
@@ -25,6 +25,9 @@
                  :tooltip="rule.status.description" />
       </span>
       <span class="display-flex flex-direction-row align-items-center">
+        <f7-segmented v-if="!newScript && isBlockly" class="margin-right">
+          <f7-button outline small :active="blocklyShowLabels" icon-f7="square_on_circle" :icon-size="($theme.aurora) ? 20 : 22" class="no-ripple" @click="toggleBlocklyItemLabelId" :tooltip="'Toggle to show either item labels or IDs'" />
```

Comment:
... and hide the button in code preview as it is not working there (error thrown):

```suggestion
        <f7-button v-if="!newScript && isBlockly && !blocklyCodePreview" outline small :active="blocklyShowLabels" icon-f7="square_on_circle" :icon-size="($theme.aurora) ? 20 : 22" class="no-ripple margin-right" @click="toggleBlocklyItemLabelId" :tooltip="'Toggle to show either item labels or IDs'" />
```